commands/password-managers/bitwarden/copy-first-matching-password.sh

```diff
@@ -20,6 +20,10 @@
 # @raycast.author Phil Salant
 # @raycast.authorURL https://github.com/PSalant726
 # @raycast.description Search all items in a Bitwarden vault, and copy the password of the first search result to the clipboard.
+notFound() {
```

Comment:
Just a nitpick, but I'd like it if a blank line separated the documentation from the code here.
```suggestion

notFound() {
```